release.yaml

```diff
@@ -110,7 +110,7 @@ rancher-vsphere-cpi:
 rancher-vsphere-csi:
   - 104.0.0+up3.0.2-rancher1
   - 104.1.0+up3.1.2-rancher1
-  - 104.2.0+up3.1.2-rancher2
+  - 104.1.1+up3.1.2-rancher2
```

Comment:
Please, keep only the `rancher-vsphere-csi` version that you're currently working on and sill be released in the upcoming _ Rancher v2.9.0_ Release.